packages/tailwindcss/src/theme.ts

```diff
@@ -1,6 +1,8 @@
 import type { AtRule } from './ast'
 import { escape } from './utils/escape'
 
+const ESCAPED_STAR_REGEX = /-\\\*$/
```

Comment:
Not sure about the name 😄

Could potentially drop the `-` from the RegEx

Reply:
Maybe `ESCAPED_ASTERISK_REGEX` 
? 😄